tools/provision.sh

```diff
@@ -63,6 +63,7 @@ function platform_linux_main() {
   brew_tool libidn
   brew_tool libedit
   brew_tool libtool
+  brew_tool libyaml
```

Comment:
On Ubuntu 16.04, I am unable to 'make deps' without libyaml. pip install of requirements.txt is failing to link. Complaining about missing libyaml. Not sure why no one else is seeing this! I can remove this and work-around!